### ISSUE TITLE
ReplayGain Analysis is broken

### ISSUE THREAD

**Issue by uklotzde** [mixxxdj/mixxx#13702]
### Bug Description

The track property remains empty.

```
warning [AnalyzerThread 1 #2] AnalyzerEbur128::storeResults() averageLufs invalid: -10.4834
```

### Version

2.5, main

**Comment by Holzhaus** [comment]
Ah, 2.4 is also affected. Already in bed now, I'll cherry-pick the fix into 2.4 tomorrow.

**Comment by fwcd** [comment]
Stumbled upon this today. The fix seems to be in 2.5, but not main yet 🤔

**Comment by Swiftb0y** [comment]
indeed.
